packages/babel-plugin-transform-object-rest-spread/test/fixtures/object-spread/assignment/expected.js

```diff
@@ -1,5 +1,5 @@
 var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
 
-z = _extends({ x }, y);
+z = _extends({}, { x }, y);
```

Comment:
@xtuc Do you have a description of what this is fixing? I agree that it's not desirable to change this if it can be avoided. I assume this fix is unrelated to the getter issue, but hard to say without a repro case for this.